### DISCUSSION TITLE
Use Modify(false) instead of Modify() | Explicitly specify whether to run with or without validation

### DISCUSSION THREAD

**Discussion by rvanbekkum** [StefanMaron/BusinessCentral.LinterCop#111] [New Rule]
It would be nice to have a rule that checks that `Modify(false)` is used in favor of `Modify()`, i.e., be explicit and show that it is intentional to invoke a `Modify` _without_ validation.
The same applies for other methods, like `Insert`, `Delete`, ...

**Comment by Arthurvdv** [comment]
New rule LC0040 “Explicitly set the RunTrigger parameter on build-in methods.” in the [0.30.9](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.9) version of the LinterCop, thanks to @tinestaric!
